helm/fiftyone-teams-app/Chart.yaml

```diff
@@ -1,7 +1,8 @@
+---
 apiVersion: v2
 name: fiftyone-teams-app
 namespace: fiftyone-teams
-description: A FiftyOne Teams Helm Chart
+description: FiftyOne Teams is the enterprise version of the open source [FiftyOne](https://github.com/voxel51/fiftyone) project.
```

Comment:
helm-docs includes this as a template. I incorporate content from the removed `helm/README.md`.